mautic/mautic
# Segment API: category info and search by category alias

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | "features"
| Bug fix?                               | no
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | https://github.com/mautic/mautic-documentation/pull/209
| Related developer documentation PR URL | https://github.com/mautic/developer-documentation/pull/199
| Issue(s) addressed                     | Fixes #... 



#### Description:
This pull requests provides category info in segment API data and adds standard category search by query `category:category-alias`.


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Create segment with category and make API request to view data. Category info should be available `GET /api/segments/{YOUR_SEGMENT_ID}`
3. Test search by category alias though API `/api/segments?search=category:{YOUR_CATEGORY_ALIAS}`. Search can be also tested in segment list in Mautic panel by query `category:category-alias`.
